Source/UI/IdentityUI.swift

```diff
@@ -477,6 +488,105 @@ extension IdentityUI {
         }
     }
 
+    @available(iOS 13.0, *)
+    private func showSharedWebCredentialsController(client: Client,
+                                                    presentingViewController: UIViewController,
+                                                    localizedTeaserText: String?,
+                                                    scopes: [String],
+                                                    completion: @escaping (Output) -> Void) {
+        func fallback() {
+            show(input: .byLoginMethod(.password,
+                                       presentingViewController: presentingViewController,
+                                       localizedTeaserText: localizedTeaserText,
+                                       scopes: scopes),
+                 client: client,
+                 completion: completion)
+
+            sharedWebCredentialsController = nil
+        }
+
+        sharedWebCredentialsController = SharedWebCredentialsController { [weak self] result in
+            guard case .success((let username, let password)) = result,
+                  let email = EmailAddress(username),
+                  let self = self else {
+                fallback()
```

Comment:
nit: Can we log the error here? It would be good to know whether it's just because of cancellation or invalid user name (malformed e-mail)